content/posts/September-2020.md

```diff
@@ -58,3 +58,10 @@ Following bugs were fixed:
 
 - Packit dropping leading zeros in version. [packit#814](https://github.com/packit/packit/pull/814)
 - Packit CLI issue caused by picking incorrect copr project name. [packit#971](https://github.com/packit/packit/pull/971)
+
+## Week 40 (September 28th - October 2nd)
+
+- Packit-service is now explicitly checking if requested copr-build targets exist and if not, the user is informed about it. [packit-service#835](https://github.com/packit-service/packit-service/pull/835)
+- Packit requires archive root dir to update the spec file correctly. The process of getting it was improved - now is extracted directly from the archive and if failed `archive_root_dir_template` is used. Please check [docu](https://github.com/packit/packit.dev/pull/167) for more details. [packit#834](https://github.com/packit-service/packit-service/pull/834)
```

Comment:
I'd simplify this to: 

    We have improved the way how packit updates `%setup` line in a spec file - you are now able to set content of `-n` option via archive_root_dir_template config option, it defaults to ...